utils/getDataTeamsAndScores.ts

```diff
@@ -0,0 +1,31 @@
+
+type ComboDataEntry = {
+  month: string;
+  team: string;
+  inclusion_score: number;
+  [key: string]: any | any;
+}
+
+type AggregatedDataEntry = {
+  month: string;
+  [team: string]: number | string | null;
+  benchmark: number;
+}
+
+export function getDataTeamsAndScores(data: ComboDataEntry[], teams: string[], key: string): AggregatedDataEntry[] {
+  const aggregatedData: Record<string, AggregatedDataEntry> = {};
+
+  data.forEach((entry) => {
+    const { month, team: entryTeam, [key]: value } = entry;
+
+    if (!aggregatedData[month]) {
+      aggregatedData[month] = { month, benchmark: 50 };
+    }
+
+    teams.forEach((team) => {
+      aggregatedData[month][team] = entryTeam === team ? value : aggregatedData[month][team] ?? null;
```

Comment:
i have no idea what this does hahah, is any one else gonna use this? in that case you should probably make it a bit more understandable

Reply:
It is a nested loop, this is the 2nd part to this loop, and it is to re-combine the data by months. 
So it checks if a team and its score value is already added to the new object (month), where a I combine the new array so that each month from feb-okt (dont get duplicates) and the team scores is added to that specific month, 

so this checks so that the right team and its value goes in to the right object and is not duplicated. 

You could write this part, like this too. 

teams.forEach((team) => {
  const teamDataEntry = aggregatedData[month];

  if (!teamDataEntry[team] || entryTeam === team) {
    teamDataEntry[team] = entryTeam === team ? value : null;
  }
});